genestack_client/scripts/genestack_application_manager.py

```diff
@@ -394,85 +393,62 @@ def resolve_jar_file(file_path):
 
 
 def mark_as_stable(application, version, app_id_list, scope):
-    try:
-        print('Setting the application version "%s" stable for scope %s'
-              % (version, scope))
-        scope = SCOPE_DICT[scope]
-        for app_id in app_id_list:
-            sys.stdout.write('%-40s ... ' % app_id)
-            sys.stdout.flush()
-            application.invoke('markAsStable', app_id, scope, version)
-            sys.stdout.write('ok\n')
-            sys.stdout.flush()
-    except GenestackException as e:
-        sys.stderr.write('%s\n' % e.message)
-        return 1
+    print('Setting the application version "%s" stable for scope %s'
+          % (version, scope))
+    scope = SCOPE_DICT[scope]
+    for app_id in app_id_list:
+        sys.stdout.write('%-40s ... ' % app_id)
```

Comment:
Maybe we should disable `stdout` buferization in the scripts? It's quite tedious to write `sys.stdout.flush()` every time. See [this blog post](http://algorithmicallyrandom.blogspot.ru/2009/10/python-tips-and-tricks-flushing-stdout.html).